tools/hmac/README.md

```diff
@@ -0,0 +1,34 @@
+## HMAC Tools
+
+This folder contains the scripts/tool to calculate the HMAC signature of the webhook payload.  
+They can be used to troubleshoot the HMAC signature calculation.
+
+Check tools/hmac/package.json to confirm the Adyen Node API library version
+
+Note: make sure you are using the HMAC key used to generate the signature associated with the payload in the JSON file
+
+### Payments webhooks
+
+Copy the content of the webhook in the payload.json (or provide a different file), then run with: 
+`node calculateHmacPayments.js {hmacKey} {path to JSON file}`
```

Comment:
nit: Distinguish between `relativeFilePath` & `absoluteFilePath` by prefixing the `path` accordingly

Reply:
file can be either, hence keeping a generic name